tests/system/providers/amazon/aws/example_redshift.py

```diff
@@ -77,9 +77,9 @@ def create_connection(conn_id_name: str, cluster_id: str):
 @task
 def setup_security_group(sec_group_name: str, ip_permissions: list[dict]):
     client = boto3.client("ec2")
-    vpc_id = client.describe_vpcs()["Vpcs"][0]["VpcId"]
+    default_vpc = client.describe_vpcs(Filters=[{"Name": "is-default", "Values": ["true"]}])["Vpcs"][0]
```

Comment:
Would it make sense to use the existing utility to get the `vpc` as we do in the `example_dms` DAG test? https://github.com/apache/airflow/blob/64b0872d92609e2df465989062e39357eeef9dab/tests/system/providers/amazon/aws/example_dms.py#L283 

Might need some refactoring for this task to accept the VPC ID as a parameter, but I feel it's worth it.

With this, we could reuse the utilities everywhere and when we need to make any change e.g. changing the filter name value, we need to do it in one place only instead of changing it in multiple places.

Reply:
+1

Reply:
good idea, I didn't know this existed, done